proto/common.proto

```diff
@@ -201,27 +201,22 @@ message Buffs {
 		bool draenei_racial_caster = 9;
     int32 shadow_priest_dps = 10;
 
-    // Target Debuff
-    bool judgement_of_wisdom = 11;
-    bool improved_seal_of_the_crusader = 12;
-    bool misery = 13;
-
 		// Drums
-		Drums drums = 14;
+		Drums drums = 11;
 
     // Item Buffs
-    int32 atiesh_mage = 15;
-    int32 atiesh_warlock = 16;
-    bool braided_eternium_chain = 17;
-    bool eye_of_the_night = 18;
-    bool chain_of_the_twilight_owl = 19;
-    bool jade_pendant_of_blasting = 20;
+    int32 atiesh_mage = 12;
```

Comment:
you dont have to re-number protos when removing old fields... not sure if you knew that or were just doing this for cleanliness

Reply:
Yeah I'm aware, just like to to it. At google we cant because servers never stop running so it feels good.   ;)